integrations/RedditPixel/browser.js

```diff
@@ -0,0 +1,85 @@
+import logger from "../../utils/logUtil";
+
+class RedditPixel {
+  constructor(config) {
+    this.advertiserId = config.advertiserId;
+    this.name = "REDDIT_PIXEL";
+  }
+
+  init() {
+    logger.debug("===In init RedditPixel===");
+
+    !(function (w, d) {
+      if (!w.rdt) {
+        var p = (w.rdt = function () {
+          p.sendEvent
+            ? p.sendEvent.apply(p, arguments)
+            : p.callQueue.push(arguments);
+        });
+        p.callQueue = [];
+        var t = d.createElement("script");
+        (t.src = "https://www.redditstatic.com/ads/pixel.js"), (t.async = !0);
+        var s = d.getElementsByTagName("script")[0];
+        s.parentNode.insertBefore(t, s);
+      }
+    })(window, document);
+
+    rdt("init", `${this.advertiserId}`);
+  }
+
+  isLoaded() {
+    logger.debug("===In isLoaded RedditPixel===");
+    return !!(window.rdt && window.rdt.advertiserID === this.advertiserId);
+  }
+
+  isReady() {
+    logger.debug("===In isReady RedditPixel===");
```

Comment:
we should properly check once if this is okay or simply use `!!window.rdt`

Reply:
both the earlier checks are working now as expected.